dplyr_in_r.Rmd

```diff
@@ -0,0 +1,263 @@
+# dplyr pacakage in R
+
+Siyu Shen  
+
+
+The **dplyr** package is mainly used for cleaning and sorting data. By using **dplyr** package, we can efficiently process the data, which is very important for people to do data analysis. In this note, I am going to introduce the most common used functions in **dplyr**.
+
+```{r eval=FALSE, message=FALSE, warning=FALSE}
+#install 'dplyr' package
+install.packages(dplyr)
+```
```

Comment:
Please follow the submission instructions <https://jtr13.github.io/cc22tt/github-submission-instructions.html> and remove this statement